static/css/devhub/static-theme.less

```diff
@@ -16,7 +16,7 @@
         }
     }
     h3 {
-        font-size: 1.1em;
+        font-size: 12pt;
```

Comment:
`pt` is an odd choice of unit here, since we're not printing. Why the change ? I don't see anything regarding this in the issue.

Reply:
I think I meant px.  I changed it because I made `Select a header image for your theme` a `<h3>` also to get equal spacing; but as it's inside a `<label>` that has `1.1em` css rule, and em is a relative size, the `<h3>` inside the label ended up bigger than the others.

Reply:
i.e. stupid css.

Reply:
Ok, that makes sense, but definitely change it to `px` then :)

Reply:
https://github.com/mozilla/addons-server/pull/7417/commits/552784611cbae82b3b8e090a865803d24bd757cd